internal/controllers/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -661,7 +661,7 @@ func TestMachineHealthCheck_Reconcile(t *testing.T) {
 		}).Should(Equal(1))
 	})
 
-	t.Run("it marks unhealthy machines for remediation when the unhealthy Machines exceed MaxUnhealthy", func(t *testing.T) {
+	t.Run("it doesn't mark unhealthy machines for remediation when the unhealthy Machines exceed MaxUnhealthy", func(t *testing.T) {
```

Comment:
I'm not sure this is entirely correct, because it sets HealthCheckSucceeded=false, but it doesn't add OwnerRemediated. might be 
```suggestion
	t.Run("it mark unhealthy machines for remediation but it doesn't engage machine owner when the unhealthy Machines exceed MaxUnhealthy", func(t *testing.T) {
```
Same below

Reply:
I used slightly different wording, but I think now it expresses what the test is doing